lib/smart_answer/formatting_helper.rb

```diff
@@ -16,9 +16,8 @@ def format_salary(salary)
       number_to_currency(salary.amount, precision: 0) + " per " + salary.period
     end
 
-    def format_date(date)
-      return nil unless date
-      date.strftime('%e %B %Y')
+    def format_date(date, date_format = :govuk)
```

Comment:
I'm not sure we really need this helper.

We have configured `Date::DATE_FORMATS[:govuk]` so that we are not duplicating our format strings.

`to_s(format = :default)` is essentially the same as `format_date(date, date_format = :govuk)`

By creating a `format_date` method we've encapsulated that the default date format should be `:govuk`, but I'm not sure that really gives us any value because we still have to remember to use the `format_date` method which is no different to having to remember to use `to_s(:govuk)`.

I'd favour using `to_s(:govuk)` over the extra layer of encapsulation.

Reply:
@andrewgarner yes you are right, but I tried the other approach and if we use `to_s(:govuk)` in the code, then we should manage somewhere else the `nil` case:
```
date = nil
date.to_s(:govuk)
=>  error: 1 parameter instead of 0 given to `to_s` method 
```
Apart from this I feel that `format_date` is easier to write and remember than `to_s(:govuk)`.
But let me know if you are still for the latter approach. Thanks.

Reply:
Hmm, I feel like that is telling us we should be handling the `nil` case correctly elsewhere though.